src/Altinn.AccessManagement/Utilities/IdentifierUtil.cs

```diff
@@ -152,7 +152,7 @@ public static string MaskSSN(string input)
         /// <returns>AttributeMatch model representation of the identifier</returns>
         public static AttributeMatch GetIdentifierAsAttributeMatch(string party, HttpContext context)
         {
-            if (party.Equals(OrganizationHeaderTrigger))
+            if (string.IsNullOrEmpty(party) || party.Equals(OrganizationHeaderTrigger))
```

Comment:
party should never be null or empty?

Reply:
it shouldn't, but the value can still be null. Added it so that the function can't throw an exception based on the input value